examples/example_vault_chef/libraries/secrets_management.rb

```diff
@@ -0,0 +1,113 @@
+#
+# Author:: Jeff Brimager
+# Author:: Paul Bradford (<pbradford@chef.io>)
+# Author:: Davin Taddeo (<davin@davintaddeo.com>)
+# Cookbook:: chef_magic
+# Library:: secrets_management
+#
+# Copyright:: 2019-2020, Davin Taddeo
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+require 'vault'
+require 'tomlrb'
+
+module ChefMagic
```

Comment:
This module name should match the cookbook name (at least by generator standards) we should update that here (line 24) and on line 113.